### PR TITLE
Add RabbitMQ Export Stage

### PR DESCRIPTION
**Pull request by AdrianSantoPasqualini**
Adds new pipeline stage to export the news article jsons
Adds abstract method `destroy` to the pipeline stage which can be overwritten to perform any cleanup needed (here it is used to close the rabbitmq connection).

Power Rabbit

### CODE REVIEW

File: pipeline/export_manager.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import logging
```
Comment (arjun-vasudevan) [inline pipeline/export_manager.py:1]: 4 spaces for indentation smh

Comment (AdrianSantoPasqualini) [reply pipeline/export_manager.py:1]: Done

File: pipeline/pipeline_stage.py [hunk 2]
```diff
@@ -44,3 +45,8 @@ def start(self):
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def _post_process(self):
+        """
+        Perform any needed cleanup for the stage
+        """
```
Comment (susheelk) [inline pipeline/pipeline_stage.py:52]: You need to raise `NotImplementedError` or add a `pass` here. I don't think you can have an empty function body in python

**Review by susheelk — CHANGES_REQUESTED** [review]
# +1
Resolve the comments and merge at your discretion :+1:

File: pipeline/pipeline_stage.py (hunk 2 above)
Comment (AdrianSantoPasqualini) [reply pipeline/pipeline_stage.py:52]: Mine worked even without this, but you're right we should have this for clarity and consistency

**Pull request merged**